packages/noports_core/lib/src/common/file_system_utils.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:io';
```

Comment:
This was utils.dart, but I extracted the minimum required shared file utilities.